### ISSUE TITLE
Runepriest's Master rune of fury

### ISSUE THREAD

**Issue by barfyman92** [WarEmu/WarBugs#14899]
Issues should be unique. Check if someone else reported 
the issue first, and please don't report duplicates.
Only ONE issue in a report. Don't forget screens or a video.

**Expected behavior and actual behavior:**

It says 20% procc rate, it currently proccs way more than that. Closer to 90% or more

**Steps to reproduce the problem:**

Go to the testing dummies in Altdorf, apply a DoT to one of them to get combat, exhaust your AP with the costly 55 Rune of Mending, pop rune of fury and spam some Grungi's Gift to max your AP in a few GCD's

Or keep spamming Rune of Mending at full AP with Rune of Fury up, you will barely lose AP. 

**Testing Screenshots/Videos/Evidences (always needed):**

https://www.youtube.com/watch?v=r-VPh8aDGoU&feature=youtu.be

EDIT: Added Video as requested

**Comment by sioding** [comment]
**Thanks for the report.** We need as much information as you can give us for reproducing the bug. That means: when... where... what.. and please ensure that you provide the appropriate evidence, be it in screenshot or video format.

**Comment by Raebrek** [comment]
I just tested this on my RP and can confirm it is happening. It's definitely not happening with only Rune of Mending, pretty much any heal ability can be spammed and will give you back AP nearly every time. (this was with no tactics on so restorative burst was not proc'ing). Also, seems to be proc'ing on DPS abilities just as frequently so it's not just heals.

Tested on zealot and unable to reproduce the same problem.
